projects/addon-editor/components/color-selector/color-selector.template.html

```diff
@@ -69,8 +69,7 @@
             [class.t-direction_active]="isDirectionActive(button)"
             [icon]="getIcon(button)"
             [tuiHint]="button"
-            [tuiHintId]="directionBtn.id"
-            [tuiDescribedBy]="directionBtn.id"
+            [tuiHintDescribe]="''"
```

Comment:
nitpicking:

```suggestion
            tuiHintDescribe
```

Reply:
Prettier would order it to the top and cause this issue:
https://github.com/Tinkoff/taiga-ui/issues/2494

Reply:
Once we solve is, we would change all these places like you suggest